apiserver/common/modeldestroy.go

```diff
@@ -53,6 +53,10 @@ func DestroyController(
 		for _, uuid := range uuids {
 			modelSt, release, err := st.GetBackend(uuid)
 			if err != nil {
+				if errors.IsNotFound(err) {
+					// Model is already in the process of being destroyed.
+					continue
+				}
```

Comment:
Do we have any way to create a mock test for this like we did for the other occurrences ?

Reply:
added a test :)